terragrunt.hcl

```diff
@@ -30,7 +30,7 @@ remote_state {
 
   config = {
     encrypt = true
-    bucket = "kuberoke-terraform-state"
+    bucket = "kuberoke-chi-23-terraform-state"
```

Comment:
I don't think this should be the default in `main`

Reply:
OTOH this needs to be customized by every user, so we should probably add a comment here reflecting this.